From afa2009698722ab7c3e22490fd516e1ea7f097c7 Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Sat, 14 Dec 2024 10:14:45 -0500
Subject: [PATCH] Revert test adjustments from
 5ccbc62175eece867b498115724eb8d3fa27acb0

This allows the tests to keep passing with httpx 0.27.x.
---
 tests/middleware/test_wsgi.py | 2 +-
 tests/test_requests.py        | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/tests/middleware/test_wsgi.py b/tests/middleware/test_wsgi.py
index e4ac66a..58696bb 100644
--- a/tests/middleware/test_wsgi.py
+++ b/tests/middleware/test_wsgi.py
@@ -77,7 +77,7 @@ def test_wsgi_post(test_client_factory: TestClientFactory) -> None:
     client = test_client_factory(app)
     response = client.post("/", json={"example": 123})
     assert response.status_code == 200
-    assert response.text == '{"example":123}'
+    assert response.text == '{"example": 123}'
 
 
 def test_wsgi_exception(test_client_factory: TestClientFactory) -> None:
diff --git a/tests/test_requests.py b/tests/test_requests.py
index 665dceb..f0494e7 100644
--- a/tests/test_requests.py
+++ b/tests/test_requests.py
@@ -91,7 +91,7 @@ def test_request_body(test_client_factory: TestClientFactory) -> None:
     assert response.json() == {"body": ""}
 
     response = client.post("/", json={"a": "123"})
-    assert response.json() == {"body": '{"a":"123"}'}
+    assert response.json() == {"body": '{"a": "123"}'}
 
     response = client.post("/", data="abc")  # type: ignore
     assert response.json() == {"body": "abc"}
@@ -112,7 +112,7 @@ def test_request_stream(test_client_factory: TestClientFactory) -> None:
     assert response.json() == {"body": ""}
 
     response = client.post("/", json={"a": "123"})
-    assert response.json() == {"body": '{"a":"123"}'}
+    assert response.json() == {"body": '{"a": "123"}'}
 
     response = client.post("/", data="abc")  # type: ignore
     assert response.json() == {"body": "abc"}
-- 
2.47.1

